pallets/runtime/tests/src/confidential_test.rs

```diff
@@ -14,6 +14,17 @@ use test_client::AccountKeyring;
 type Asset = asset::Module<TestStorage>;
 type Confidential = confidential::Module<TestStorage>;
 type Origin = <TestStorage as frame_system::Trait>::Origin;
+use cryptography::{
+    asset_proofs::{CommitmentWitness, ElgamalSecretKey},
+    mercat::{
+        account::{convert_asset_ids, AccountCreator},
+        AccountCreatorInitializer, EncryptionKeys, PubAccountTx, SecAccount,
+    },
+    AssetId,
+};
+use curve25519_dalek::scalar::Scalar;
+use rand::{rngs::StdRng, SeedableRng};
+use schnorrkel::{ExpansionMode, MiniSecretKey};
```

Comment:
In the final version of this PR, these two will most likely be removed. More on this in the comment below.